java_gen/templates/custom/OFStatV6.Builder.java

```diff
@@ -1,3 +1,17 @@
+    private OFOxsList.Builder oxsFieldsBuilder;
+
+    private void initBuilder() {
+        if (oxsFieldsBuilder != null)
+            return;
+        oxsFieldsBuilder = new OFOxsList.Builder();
+    }
+
+    private void updateOxsList() {
+        this.oxsFields = this.oxsFieldsBuilder.build();
+        this.oxsFieldsSet = true;
```

Comment:
It's an *immutable* map, so to be able to add fields, we need to keep a builder representation. This method updates the immutable map after the builder has been updated. It's the exact analogue of the code for the [Match Builder](https://github.com/floodlight/loxigen/blob/master/java_gen/templates/custom/OFMatchV3.Builder.java#L10).

I agree this is kind-of ugly; it would be cleaner to either have *only* an OFOxsList or a builder. The reason is that `OFOxsList` part is auto-generated by Loxigen. So we just add a builder and make sure we synchronize the two.